src/scss/_global.scss

```diff
@@ -9,3 +12,17 @@
   background-position: center center;
   background-repeat: no-repeat;
 }
+
+// REMOVES THE STATEMENT ABOUT NO CONTENT
+#first-time { display: none; }
+
+button {
+  cursor: pointer;
+  color: color("white");
+  width: 60%;
+  padding: 5px 0;
+  background-color: color("red");
+  border: 2px outset color("red");
+  border-radius: 15px;
+  font-size: 1.5em;
+}
```

Comment:
We can discuss, but in general I've found it's best to have a `_buttons` partial and tuck away button styling into that. Sometimes the buttons styling can get more complex, and also makes it easier to find if it's in it's own file.

Be sure to add the additional import for the buttons partial to main.scss.